tests/test_function_utils.py

```diff
@@ -32,3 +33,50 @@ def boo(param_1, param_2):
         key6 = build_value_by_func_parameters('xyz_{param_1}_{param_2}', boo, None, None)
         self.assertNotEqual(key6, None)
         self.assertEqual(key6, 'xyz_None_None')
+
+    def test_func_param_to_dict(self):
+        def boo(param_1, param_2):
```

Comment:
give it a better name please `params_test_function` or something more meaningful